students/Hovhannes_Muradyan/Lesson1/hw1.txt

```diff
@@ -0,0 +1,99 @@
+Задача 1.
+С помощью битовых операций проверить число на четность 
+примеры 7 -> false, 4578 -> true
+bool is_even(int x)
+{
+	return !(x&1);
+}
+
+
+
+
+
+
+
+Задача 2.
+Посчитать количество нулей в начале двоичного представления:
+пример x = 784252715 (00101110101111101011111100101011) -> функция должна возвращать 2
+int nulls_count(int x)
+{
+  int counter = 0;
+  for(int i = 30; i >=0 ; i--){
```

Comment:
using hardcoded values like `int i = 30` is not a good practice in C++, what if sizeof(int)*8 != 32 ? So a better generic solution would be `int i = sizeof(int)*8-2`. This is a small remark, but in C++ code it's important to not use hard-coded values.